### ISSUE TITLE
NodeJS (20.x) Change distribution repository

### ISSUE THREAD

**Issue by JeGoi** [geerlingguy/ansible-role-nodejs#154]
Source: https://github.com/nodesource/distributions
```
We'd like to inform you of important changes to our distribution repository [nodesource/distributions](https://github.com/nodesource/distributions).

What's New:

Package Changes: DEB and RPM packages are now available under the nodistro codename. We no longer package the installer coupled to specific versions. This means you can install Node.js on almost any distro that meets the minimum requirements.
Installation Scripts: The installation scripts setup_XX.x are no longer supported and are not needed anymore, as the installation process is straightforward for any RPM and DEB distro.
Questions and concerns: To resolve questions and discuss concerns about this update we've opened this discussion space [New distribution's packages](https://github.com/nodesource/distributions/discussions/#123456)
```
Need to fix repos in:
https://github.com/geerlingguy/ansible-role-nodejs/blob/master/tasks/setup-Debian.yml
and
https://github.com/geerlingguy/ansible-role-nodejs/blob/master/tasks/setup-RedHat.yml

Thanks

**Comment by zane-carter** [comment]
@geerlingguy 
Are you able to look into this?

**Comment by neilyoung** [comment]
Ansible script fails to install if system is 20.04 an nodejs_version = "20.x"

**Comment by geerlingguy** [comment]
Fixed with #155 above
